tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -2587,7 +2587,14 @@ void MklLayoutRewritePass::CopyAttrsConvCheckConstFilter(const Node* orig_node,
   // Check and set filter attribute.
   Node* filter_node = nullptr;
   TF_CHECK_OK(orig_node->input_node(1, &filter_node));
-  nb->Attr("is_filter_const", filter_node->IsConstant());
+
+  bool is_filter_const = false;
+  if (HasNodeAttr(orig_node->def(), "is_filter_const")) {
+    GetNodeAttr(orig_node->def(), "is_filter_const", &is_filter_const);
+  }
+  if (!is_filter_const) {
+    nb->Attr("is_filter_const", filter_node->IsConstant());
+  }
```

Comment:
So this will only set `is_filter_const` in the new node when the original node has the attribute set to false. Why is that? Please explain the error this is trying to fix.

Reply:
@penpornk 

This piece of new code is intended to handle different default setting of is_filter_const attribute
          FP32: default value is false
          INT8: default value is true

The original code author (Amin) in our team is on vacation this week. We will give you a firm reply next week.

Thanks!

Reply:
@penpornk Made code change after consulting with multiple teammates and testing.  

Thank you very much!